openmdao/core/tests/test_check_derivs.py

```diff
@@ -240,7 +241,10 @@ def test_component_no_check_partials(self):
         # disable partials on comp2
         #
         comp2._no_check_partials = True
-        data = prob.check_partials(out_stream=None)
+
+        # Make check_partials think we're not on CI so we'll get the expected  non-CI behavior
+        with set_env_vars_context(GITHUB_ACTION='0', TRAVIS='0'):
```

Comment:
I think we can just use the environment variable `CI` here..  it's boolean and available on GH Actions, Travis-CI and CircleCI:
https://docs.github.com/en/actions/learn-github-actions/environment-variables
https://docs.travis-ci.com/user/environment-variables
https://circleci.com/docs/env-vars